PURL-TYPES.rst

```diff
@@ -73,6 +73,20 @@ bitbucket
 
       pkg:bitbucket/birkenfeld/pygments-main@244fd47e07d1014f0aed9c
 
+bitnami
+-------
+``bitnami`` for Bitnami-based packages:
+
+- The default repository is ``https://downloads.bitnami.com/files/stacksmith``.
+- The ``name`` is the component name. It must be lowercased.
+- The ``version`` is the full Bitnami package version, including version, revision and architecture.
+- The ``arch`` is the qualifiers key for a package architecture. Available values: ``amd64`` (default) and ``arm64``.
```

Comment:
Is this documented on bitnami anywhere?

Reply:
We don't have it documented in our [docs site](https://docs.bitnami.com/) yet. Can you confirm if that is a requirement?

Reply:
Not a requirement. However, if questions arise in the future, we'll need a reference to point the user to. So yes, it should be externally documented somewhere.